NumberBaseball/Source.c

```diff
@@ -0,0 +1,150 @@
+#include <stdio.h>
+#include <stdlib.h> 
+#include<time.h> 
+
+#define LINESIZE 1024
+#define NUMBER_OF_GUESSES 3
+#define STRIKES_TO_END 3
+
+int numStrikes;
+int numBalls;
+int numOuts;
+int resultStrikes;
+int resultBalls;
+int resultOuts;
+int n1;
+int n2;
+int n3;
+int random1;
+int random2;
+int random3;
+
+void Init() {
+	numStrikes = 0;
+	numBalls = 0;
+	numOuts = 0;
+	resultStrikes = 0;
+	resultBalls = 0;
+	resultOuts = 0;
+	printf("Init\n");
```

Comment:
I think This kind of message is for developers unless messages like "Loading".
Users shouldn't see anything not related to game.

Reply:
Whoops left this in by accident. Deleted.